src/user-event/press/press.ts

```diff
@@ -73,18 +75,14 @@ const emitPressablePressEvents = async (
 
   dispatchEvent(element, 'responderGrant', EventBuilder.Common.responderGrant());
 
-  await wait(config, options.duration);
+  // We apply minimum press duration here to ensure that `press` events are emitted after `pressOut`.
+  // Otherwise, pressables would emit them in the reverse order, which in reality happens only for
+  // very short presses (< 130ms) and contradicts the React Native docs.
+  // See: https://reactnative.dev/docs/pressable#onpress
+  let duration = Math.max(options.duration, DEFAULT_MIN_PRESS_DURATION);
+  await wait(config, duration);
```

Comment:
this will work but I think the previous implementation was closer to the reality and this feels kind of like a work around to me. Wouldn't it be easier to keep the implementation of `basePress` untouched and change the duration in `press` from 0 to `DEFAULT_MIN_PRESS_DURATION`? If someone did a longPress with a duration bellow 130ms (even though it may not be realistic) pressout should come out after press and it won't be the case with this change

Reply:
Makes sense for Pressables, but we would have to keep the Text/Input as we simulate pressability behavior there